<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>728</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,75">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QListWidget" name="lst_func">
          <item>
           <property name="text">
            <string>insertion</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>selection</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_run">
          <property name="text">
           <string>Sort !</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="lbl_visu">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btn_ini">
            <property name="text">
             <string>Initial State</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_step_b">
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lbl_step">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_step_f">
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_fin">
            <property name="text">
             <string>Final State</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>912</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_run</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slt_sort_run()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>138</x>
     <y>685</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_step_b</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slt_prev_step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_step_f</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slt_next_step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>795</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_ini</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slt_init_state()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_fin</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>slt_fin_state()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>838</x>
     <y>607</y>
    </hint>
    <hint type="destinationlabel">
     <x>455</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slt_sort_run()</slot>
  <slot>slt_prev_step()</slot>
  <slot>slt_next_step()</slot>
  <slot>slt_init_state()</slot>
  <slot>slt_fin_state()</slot>
 </slots>
</ui>
